ping/index.test.js

```diff
@@ -98,6 +98,33 @@ it('sends only one ping if timeout is bigger than ping', async () => {
   expect(test.leftSent).toEqual([['ping', 1]])
 })
 
+it('not try clear timeout if it does not set', async () => {
```

Comment:
We need an English sentence here. For instance `it('do not try to clear`

Reply:
Fixed. Thanks :)